src/Algebra/Graph/AdjacencyIntMap/Algorithm.hs

```diff
@@ -16,12 +18,13 @@
 module Algebra.Graph.AdjacencyIntMap.Algorithm (
     -- * Algorithms
     dfsForest, dfsForestFrom, dfs, reachable, topSort, isAcyclic,
-
+    bfsForest, bfsForestFrom, bfs, 
```

Comment:
A general comment: please put `bfs*` in front of `dfs*` throughout your PR. They are quite similar and `bfs` should probably get ahead simply lexicographically. I'm happy to reorder things too, but then I'd probably end up touching almost every line in this PR!

Reply:
Should the bfs code come before in the module as well? I changed the export lists for now

Reply:
Yes, I prefer to provide implementations in the same order as they appear in the export list. Same for `API` methods.